src/components/DiffView/index.spec.tsx

```diff
@@ -2,16 +2,7 @@
 import queryString from 'query-string';
 import React from 'react';
 import { Alert } from 'react-bootstrap';
-import {
-  Diff,
-  HunkInfo,
-  Hunks,
-  RenderGutterParams,
-  WidgetMap,
-  parseDiff,
-  getChangeKey,
-  tokenize,
-} from 'react-diff-view';
+import * as ReactDiffView from 'react-diff-view';
```

Comment:
I am fine with this change but I am wondering why it is needed.

Reply:
I needed to do this to get `jest.spyOn(ReactDiffView, 'tokenize')` to work. I checked docs and Googled quite a bit and couldn't find a way to use `spyOn` with just an imported function, as opposed to having to specify the imported module and function.